it/test/connectors/ReferenceDataConnectorSpec.scala

```diff
@@ -244,20 +255,24 @@ class ReferenceDataConnectorSpec extends ItSpecBase with ScalaCheckPropertyCheck
             .willReturn(okJson(customsOfficesResponseJson))
         )
 
-        val expectedResult = NonEmptyList(
+        val expectedResult = NonEmptySet.of(
           CustomsOffice("GBtestId1", Some("testName1"), Some("testPhoneNumber")),
-          List(CustomsOffice("GBtestId2", Some("testName2"), None))
+          CustomsOffice("GBtestId2", Some("testName2"), None)
         )
 
-        connector.getCustomsOfficesForCountry(countryCode).futureValue mustBe expectedResult
+        connector.getCustomsOfficesForCountry(countryIds: _*).futureValue mustBe expectedResult
       }
 
       "must throw a NoReferenceDataFoundException for an empty response" in {
-        checkNoReferenceDataFoundResponse(url, connector.getCustomsOfficesForCountry(countryCode))
+        val countryId = "AR"
+        val url       = s"/$baseUrl/lists/CustomsOffices?data.countryId=AR&data.roles.role=DES"
+        checkNoReferenceDataFoundResponse(url, connector.getCustomsOfficesForCountry(countryId))
       }
 
       "must return an exception when an error response is returned" in {
-        checkErrorResponse(url, connector.getCustomsOfficesForCountry(countryCode))
+        val countryId = "GB"
```

Comment:
could this use the Constants file

Reply:
Going to leave as is if that's ok. It's largely an arbitrary value